frontend/src/Components/Form/EnhancedSelectInput.js

```diff
@@ -28,6 +28,18 @@ function getSelectedOption(selectedIndex, values) {
   return values[selectedIndex];
 }
 
+function getMinId(values) {
```

Comment:
This can be simplified to `values.reduce((min, v) => v.key < min ? v.key : min, values[0].key)`